src/postwidder.jl

```diff
@@ -0,0 +1,90 @@
+# Gaver-Stehfest method "Post-Widder"
+
+# 1.
+# Stehfest, Harald.
+# Algorithm 368: Numerical inversion of Laplace transforms [D5].
+# Communications of the ACM 13.1 (1970): 47-49.
+
+# 2.
+# Kuhlman, Kristopher L. 
+# Review of inverse Laplace transform algorithms for Laplace-space numerical approaches.
+# Numerical Algorithms 63.2 (2013): 339-355.
+
+# 3. 
+# Al-Shuaibi, Abdulaziz. 
+# Inversion of the laplace transform via post—widder formula.
+# Integral Transforms and Special Functions 11.3 (2001): 225-232.
+
+
+#= Compute Post-Widder coefficients Vk for N terms. Also referred to as Salzer summation weights or Stehfest coefficients.
+Only depend on the approximation order (N) and the precision. M must be even but can be precomputed and used only once. 
+Coefficients get very large in magnitude and oscillate rapidly. Must be careful to avoid catastrophic cancellation for lower precision. 
+For  N < 18 double precision is usually ok but less accurate, utilizng defualt BigFloat precision and N = 36 usually provides sufficient accuracy.
+=#
+function _PWcoeffs(N)
+    if isodd(N)
+        N += 1
+        @warn "N must be even... increment N += 1"
+    end
+    v = zeros(BigFloat, N)
+    aux = zero(eltype(v))
+    for k in 1:N
+        for j in floor(Int, (k + 1)/2):minimum(Int, [k, N/2])
```

Comment:
It's probably best to use `div(N, 2)` or `N ÷ 2` in these places. Maybe also `_PWcoeffs(N::Integer)`.

Reply:
Not a problem I went through and made these changes in a few places I saw.

Reply:
Looking through the julia docs there is a built in function for floor integer division which shortens this line. I've updated `floor(Int, (k + 1)/2)` to `fld(k + 1, 2)`